source/Calamari.Common/Features/Substitutions/ISubstituteInFiles.cs

```diff
@@ -6,6 +6,7 @@ namespace Calamari.Common.Features.Substitutions
     public interface ISubstituteInFiles
     {
         void SubstituteBasedSettingsInSuppliedVariables(string currentDirectory);
+        void SubstituteBasedSettingsInSuppliedVariables(string currentDirectory, string[] filesToTarget);
```

Comment:
Do we really need another one, this one is pretty much the same as `Substitute` method below?
If the issue is the name, we could just have a extension method?

Reply:
Good call.

Reply:
Fixed